indexing-engine/what-if-analysis.mdx

```diff
@@ -51,5 +51,5 @@ You can read more behind the scenes the details in our blog post,
 [How we deconstructed the Postgres planner to find indexing insights](/blog/deconstructing-the-postgres-planner).
 
 After "What if?" analysis has completed, we utilize the
-["Good Enough" Index Selection Algorithm](/docs/indexing-engine/good-enough-index-selection)
+[Index Selection](/docs/indexing-engine/good-enough-index-selection) model
```

Comment:
bad link? /docs/indexing-engine/good-enough-index-selection is gone

Reply:
Well, technically there is a redirect in https://github.com/pganalyze/pganalyze-web/pull/509 so I guess it's fine as is